docs/en/writing-running-appium/caps.md

```diff
@@ -71,7 +71,7 @@ These Capabilities are available only on Android-based drivers (like
 |`androidInstallTimeout`|Timeout in milliseconds used to wait for an apk to install to the device. Defaults to `90000` |e.g., `90000`|
 |`androidInstallPath`| The name of the directory on the device in which the apk will be push before install. Defaults to `/data/local/tmp` |e.g. `/sdcard/Downloads/`|
 |`adbPort`|Port used to connect to the ADB server (default `5037`)|`5037`|
-|`systemPort` | `systemPort` used to connect to [appium-uiautomator2-driver](https://github.com/appium/appium-uiautomator2-driver), default `4724` | e.g., `4724` |
+|`systemPort` | `systemPort` used to connect to [appium-uiautomator2-driver](https://github.com/appium/appium-uiautomator2-driver), default `4724`. When you run tests in parallel, you must adjust the port to avoid conflict. Read [parallel-tests](https://github.com/appium/appium/blob/master/docs/en/advanced-concepts/parallel-tests.md#parallel-android-tests) for more details. | e.g., `4724` |
```

Comment:
`to avoid a conflict` or `to avoid conflicts`

Read Parallel Testing Setup Guide ...